test/posts.test.js

```diff
@@ -70,7 +70,7 @@ describe('test /posts/:id responses', () => {
   const missingGuid = 'http://missing-guid';
   const randomGuid = 'http://random-guid';
 
-  const feed1 = new Feed('Feed Author', 'http://feed-url.com/', null, null);
+  const feed1 = new Feed('Feed Author', 'http://feed-url.com/', '', null, null);
```

Comment:
I don't think we want the empty string.  It will either be `null` or a full string.  Let's not introduce this case.

Reply:
Fixed.